selectrum.el

```diff
@@ -42,6 +42,7 @@
 (require 'regexp-opt)
 (require 'seq)
 (require 'subr-x)
+(require 'minibuf-eldef)
```

Comment:
Can this be kept alphabetized?

Reply:
Ups, I did not notice that you keep them in alphabetical order. Fixed!